apps/src/templates/PauseButton.jsx

```diff
@@ -34,6 +34,12 @@ class PauseButton extends React.Component {
       ...(this.props.isPaused && styles.pausedColor),
       ...(this.props.marginRight && {marginRight: this.props.marginRight})
     };
+    const iconStyle = {
+      ...styles.icon,
+      ...(this.props.isAttached && {color: color.lighter_gray}),
+      ...(this.props.isRunning && {color: color.cyan}),
+      ...(this.props.isPaused && {color: color.orange})
```

Comment:
Can we use `styles.` instead of specific colors, to be consistent with what we do to generate `buttonStyle` just above?

Reply:
Done.